README.md

```diff
@@ -71,11 +72,18 @@ Fields-of-The-World
 ### Create Conda/Mamba environment
```

Comment:
Should we be able to remove these when #36 lands (or when we get all the repos)? My hope is that we could have someone just do 'pip install ftw-tools', and they don't need to create their environment with conda/mamba.

Reply:
Probably not because we can't install GDAL with libgdal-arrow-parquet via pip and some users (like my Windows machine) don't have them installed. So we should still offer this option...